ccds/hook_utils/dependencies.py

```diff
@@ -40,6 +42,15 @@ def write_dependencies(
             f.write("\n".join(lines))
             f.write("\n")
 
+    elif dependencies == "pyproject.toml":
+        with open(dependencies, "r") as f:
+            doc = tomlkit.parse(f.read())
+        doc["project"].add("dependencies", sorted(packages))
+        doc["project"]["dependencies"].multiline(True)
+
+        with open(dependencies, "w") as f:
+            f.write(tomlkit.dumps(doc))
+
```

Comment:
Only thing worth noting here is that folks using 2.1.0 with `-c master` will get a silent failure populating the pyproject.toml with dependencies. This code block won't exist for them and they won't get a warning about upgrading to 2.2.